.github/scripts/release-workflow-package-push.sh

```diff
@@ -35,7 +36,7 @@ while IFS="" read -r -d "" buildpack_toml_path; do
 		echo "::set-output name=id::${buildpack_id}"
 		echo "::set-output name=version::${buildpack_version}"
 		echo "::set-output name=path::${buildpack_path}"
-		echo "::set-output name=address::${buildpack_docker_repository}@$(crane digest "${image_name}")"
+		echo "::set-output name=address::docker://${image_name}"
```

Comment:
I had something similar before, it messed up things with `pack` and the registry. I ended up keeping the `address` as-is and added a `docker://` prefix where needed. Have you tried releasing after these changes?

Reply:
I haven't tested yet, but I will change it and test it

Reply:
Can I suggest diffing with https://github.com/heroku/buildpacks-jvm/commit/14889c54e0a0b598d400a6126adc95fd937e33ec? It looks like some of the code here is based on an older version that I since then fixed. I know for sure that the changes work in the commit above.

Reply:
done 👍🏼